specification/types.md

```diff
@@ -135,6 +135,7 @@ An enumeration of possible provider states.
 | READY        | The provider has been initialized, and is able to reliably resolve flag values.                    |
 | ERROR        | The provider is initialized but is not able to reliably resolve flag values.                       |
 | STALE        | The provider's cached state is no longer valid and may not be up-to-date with the source of truth. |
+| FATAL        | The provider has entered an irrecoverable error state.                                             |
```

Comment:
This is mentioned already in [1.7.1](https://github.com/open-feature/spec/blob/ae286cf5192b4e341ea691caee6a4c95582f32ed/specification.json#L291-L296), but needs inclusion here.